sql/sql_view.cc

```diff
@@ -432,6 +432,9 @@ bool mysql_create_view(THD *thd, TABLE_LIST *views,
     goto err;
 
   lex->link_first_table_back(view, link_to_local);
+
+  WSREP_TO_ISOLATION_BEGIN(WSREP_MYSQL_DB, NULL, NULL)
```

Comment:
Just cross-check that if the existing code is invoked with last params as NULL then this is okay but if existing code is invoked with last 2 param or 1 param as table value then we need to get same string here.

WSREP_TO_ISOLATION_BEGIN(WSREP_MYSQL_DB, NULL, NULL)

* Not that this is wrong but a safe check as it can mess lot of things.
* Validate this check or point for each TO ISOLATION call added.

Reply:
Most of the time it is called with (WSREP_MYSQL_DB, NULL, NULL).  The only time a parameter is passed is for CREATE_TRIGGER.